cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -69,25 +69,27 @@ def wait_for_ansible(appliance):
     appliance.wait_for_embedded_ansible()
 
 
-@pytest.yield_fixture(scope="module")
-def repository(wait_for_ansible):
-    repositories = RepositoryCollection()
+@pytest.yield_fixture(scope='function')
```

Comment:
Scope mismatch is here, it should be kept as `module`. You cannot use this fixture in "module" scoped fixture catalog_item.